external/iplib/_dummy_thread.py

```
@@ -0,0 +1,155 @@
+    """
+    if type(args) != type(tuple()):
+        raise TypeError("2nd arg must be a tuple")
+    if type(kwargs) != type(dict()):
+        raise TypeError("3rd arg must be a dict")
+    global _main
+    _main = False
+    try:
+        function(*args, **kwargs)
+    except SystemExit:
+        pass
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)